pkg/scheduler/apis/config/v1/conversion.go

```diff
@@ -71,16 +75,20 @@ func Convert_config_Extender_To_v1_LegacyExtender(in *config.Extender, out *v1.L
 		if err := Convert_config_ExtenderTLSConfig_To_v1_ExtenderTLSConfig(in.TLSConfig, out.TLSConfig, s); err != nil {
 			return err
 		}
+	} else {
+		out.TLSConfig = nil
 	}
 
-	if len(in.ManagedResources) > 0 {
+	if in.ManagedResources != nil {
```

Comment:
is this what generated code does? Although, I expect no issues with this implementation.

Reply:
yeah, it does something equivalent to this.